posts/2023/scipy23.md

```diff
@@ -0,0 +1,54 @@
+
+---
+author: Elena Romashkova
+date: 2023-08-28
+---
+
+
+# ESDS at SciPy 2023
+
+ESDS had a sizable presence at [SciPy 2023](https://www.scipy2023.scipy.org/) this year! Many of us shared our work in a formal capacity through tutorials, talks, and posters–most of these were recorded and have resources available publicly online, which we’ve compiled below. There were also lots of opportunities to make connections with people doing work in the broader scientific Python ecosystem outside of our organization. It was very cool to see lots of different tools, projects, and workflows.
+
+The [conference proceedings](https://conference.scipy.org/proceedings/) from this and prior years are now published as well.  You can find published slides, posters, and articles for some of the conference’s presentations at the link above.
+
+## ESDS and other NCAR/UCAR contributions:
+
+### Tutorials
+
+#### Xarray: Friendly, Interactive, and Scalable Scientific Data Analysis
+ *Deepak Cherian, Tom Nicholas, Negin Sobhani, Scott Henderson, Anderson Banihirwe, Jessica Scheick, Don Setiawan*
+
+- [Tutorial content](https://tutorial.xarray.dev/workshops/scipy2023/README.html)
+
+### Talks
+
+####  Better (Open Source) Homes and Gardens with Project Pythia
+*Kevin Tyle, Drew Camron*
+
+- [Recording link](https://www.youtube.com/watch?v=5ggKEmNn79c)
+- [Slides](https://zenodo.org/record/8221039 )
+
+#### Climate Model Evaluation Workflow Built on Jupyter Notebooks
+*Elena Romashkova*
+
+- [Recording link](https://www.youtube.com/watch?v=x_InyP5bUkI)
+- [Slides](https://docs.google.com/presentation/d/1jNhjVeQmTFPikWcSXX8YfylChmGTtE38/edit?usp=sharing&ouid=106991755025280121769&rtpof=true&sd=true)
+
+#### Building MetPy for the Long Term: Working to Keep an Open Source Project Sustainable
+*Ryan May*
+
+- [Recording link](https://www.youtube.com/watch?v=7hbGHHaRzBA)
+- [Slides](https://docs.google.com/presentation/d/14Db0NtTWW9P9TAlO8-_Ls_r9Hotq6Jo8tjaFJMP_lfo/edit?usp=sharing)
+
+#### Tidy Geospatial Cubes
+*Emma Marshall, Deepak Cherian, Scott Henderson*
+
```

Comment:
Checking if recording and slides are not available for this talk

Reply:
Nope, they're not - will update if they become available